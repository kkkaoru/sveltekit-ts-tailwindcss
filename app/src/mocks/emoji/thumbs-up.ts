export const mockEmojiThumbsUpSvgUrl = 'https://twemoji.maxcdn.com/v/latest/svg/1f44d.svg';
