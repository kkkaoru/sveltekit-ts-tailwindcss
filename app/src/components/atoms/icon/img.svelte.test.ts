import { render } from '@testing-library/svelte';
import { mockEmojiThumbsUpSvgUrl } from '@/mocks/emoji/thumbs-up';
import Icon from './img.svelte';

describe('shows proper heading when rendered', () => {
  it('should be matched snapshot', () => {
    const { container } = render(Icon, {
      src: mockEmojiThumbsUpSvgUrl,
      alt: 'test',
      width: 128,
      height: 128,
    });
    expect(container).toMatchSnapshot();
  });
});
