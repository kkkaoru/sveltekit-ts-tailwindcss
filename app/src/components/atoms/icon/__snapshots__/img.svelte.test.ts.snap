// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows proper heading when rendered should be matched snapshot 1`] = `
<body>
  <div>
    <img
      alt="test"
      class="w-32"
      decoding="async"
      height="128"
      loading="lazy"
      src="https://twemoji.maxcdn.com/v/latest/svg/1f44d.svg"
      width="128"
    />
  </div>
</body>
`;
